main.py

```diff
@@ -44,7 +43,7 @@ def main():
         exchange=os.getenv('RABBIT_EXCHANGE', 'test'),
```

Comment:
Any reason you didn't put these in the config like everything else?

Reply:
Will leave as this is the only place these get called